policyengine_canada/parameters/gov/provinces/nl/tax/income/supplement/income_supplement/additional_amount/max_amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Newfoundland and Labrador provides the following income supplement maximum additional amount.
+values:
+  2022-12-31: 266
```

Comment:
could you add the 2022 value and reference as well

Reply:
I did not find any info about 2022 value, and this amount I think refers to the phase out max amount, not the max amount of income supplement one can receive.